src/mscorlib/corefx/System/Globalization/CultureData.Windows.cs

```diff
@@ -567,5 +567,25 @@ private static string LCIDToLocaleName(int culture)
         {
             throw new NotImplementedException();
         }
+        
+        private static int GetAnsiCodePage(string cultureName)
+        {
+            throw new NotImplementedException();
```

Comment:
Any reason we are throwing NotImplementedExceptions for these instead of PlatformNotSupportedException? To me NotImplementedException should only be used if just haven't gotten to implementing them yet but we plan to do so which means we should have a tracking issues for it. If that is the case then I'm fine with NotImplementedException